test/spec.yml

```diff
@@ -179,23 +179,9 @@
 - test: Test deploy with TTL (number)
   setup:
     - export FUNC_NAME=ttl_numeric$RANDOM$RANDOM
-    - bn create node8 $FUNC_NAME
+    - bn create --config.ttl 10 node8 $FUNC_NAME
```

Comment:
So... now we have all these tests that `create --config.<something>` works, but not that actually having _any_ config value in `binaris.yml` works.  (No biggie, but still a change.)